### PR TITLE
[issue-599] changes for the merge clean up.

### PR DESCRIPTION
**Pull request by ravikiran23**
// added the compacted segment info in to the status file.

### CODE REVIEW

File: integration/spark/src/main/java/org/carbondata/spark/merger/CarbonDataMergerUtil.java [hunk 1]
```diff
@@ -145,171 +144,70 @@ public static String getMergedLoadName(List<LoadMetadataDetails> segmentsToBeMer
...
 
-  private static boolean checkIfOldMergeLoadCanBeDeleted(CarbonFile eachMergeLoadFolder,
-      LoadMetadataDetails[] details) {
-    boolean found = false;
-    for (LoadMetadataDetails loadDetail : details) {
-      if (null != loadDetail.getMergedLoadName() && (CarbonCommonConstants.LOAD_FOLDER + loadDetail
-          .getMergedLoadName()).equalsIgnoreCase(eachMergeLoadFolder.getName())) {
-        found = true;
-        break;
+    for (LoadMetadataDetails loadDetail : loadDetails) {
```
Comment (gvramana) [inline integration/spark/src/main/java/org/carbondata/spark/merger/CarbonDataMergerUtil.java]: get time stamp outside and apply to all

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala [hunk 2]
```diff
@@ -381,9 +385,17 @@ object CarbonDataRDDFactory extends Logging {
       val convLoadDetails = carbonLoadModel.getLoadMetadataDetails.asScala
       if (convLoadDetails.nonEmpty) {
         convLoadDetails.foreach { l =>
-          val loadCount = Integer.parseInt(l.getLoadName)
-          if (currentLoadCount < loadCount) {
-            currentLoadCount = loadCount
+          var loadCount = 0
```
Comment (gvramana) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: add comment explaining case

File: integration/spark/src/main/java/org/carbondata/spark/merger/CarbonDataMergerUtil.java (hunk 1 above)
Comment (ravikiran23) [inline integration/spark/src/main/java/org/carbondata/spark/merger/CarbonDataMergerUtil.java]: Done

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala (hunk 2 above)
Comment (ravikiran23) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: added comments

**Pull request merged**